ports/esp32/boards/GENERIC/mpconfigboard.h

```diff
@@ -0,0 +1,2 @@
+#define MICROPY_HW_BOARD_NAME "GENERIC"
```

Comment:
I think it'd be good to retain the existing name of "ESP32 module". a bit more user friendly.

OTOH using GENERIC signifies that the build is a generic one and then the user knows exactly where to locate the build settings for the build (ie in the boards/GENERIC dir).

Reply:
Agreed. I've changed it but kept the dir as `GENERIC`. A quick `grep` for "ESP32 module" will resolve any confusion I hope.